src/Common/tests/DefaultOptionsProviderTests.cs

```diff
@@ -0,0 +1,73 @@
+﻿// ***********************************************************************
```

Comment:
I use an emulator under VS2008. @oznetmaster uses an actual system he works on after building on VS2008. Note that this is CF 3.5.

In this case, you can't even build under NETCF because of the call to GetEnvironmentVariable.

If you don't feel like messing around with VS2008 and NETCF, that's OK. It's generally only the two of us who do those builds right now. I have an issue item to get the CI build to do netcf, but it's low priority.